docs/basic_training/executors.md

```diff
@@ -48,6 +48,36 @@ process {
 }
 ```
 
+### Submit Nextflow as a job
+
+Nextflow should not be run as a command in the login/head node of a cluster, as this node is not prepared for commands that run for too long, even if the amount of resources is negligible. Nextflow should be submitted as a job instead, and in a job node, it will submit new tasks and manage them. In a cluster using Slurm as a job scheduler, for example, you'll need to create a file similar to the one below (save it as `launch_nf.job` or any other name of your choosing):
```

Comment:
I'm not sure I entirely agree with this. I have run Nextflow on the head node for years, I think it very much depends on the cluster setup. Also, many cluster setups do not allow worker nodes to submit jobs, so the proposed solution will often not work.

```suggestion
Whilst the main Nextflow command can be launched on the login / head node of a cluster, be aware that the node must be set up for commands that run for a long time, even if the compute resources used are negligible. Another option  is to submit the main Nextflow process as a job on the cluster instead.

!!! note

    This requires your cluster configuration to allow jobs be launched from worker nodes, as Nextflow will submit new tasks and manage them from here.

For example, if your cluster uses Slurm as a job scheduler, you could create a file similar to the one below:
```

Reply:
I have also run Nextflow from the login node 😬 But it's not ideal (of course, you gotta setup worker nodes to allow the submission of jobs and stuff). Anyways, I like more your version. Committing! 😬